src/Miew.js

```diff
@@ -210,6 +210,31 @@ function _setContainerContents(container, element) {
   parent.appendChild(element);
 }
 
+Miew.prototype._updateShadowCamera = function () {
+  this._gfx.scene.updateMatrixWorld();
+  for (let i = 0; i < this._gfx.scene.children.length; i++) {
+    if (this._gfx.scene.children[i].type === 'DirectionalLight') {
+      const light = this._gfx.scene.children[i];
+      const shadowMatrix = new THREE.Matrix4().copy(light.shadow.matrix);
```

Comment:
At first please be very careful with on-the-fly object creation. The operation is expensive, so I ask you to reuse the once created variable as a temporary object to reduce using of `new` operator

Reply:
Thanks for your comment
Understood the reason and fixed this code